File: C  programs/Problems on Pattern Printing/Rows & Columns/Pattern5.c
/* Write a program which accept the number of rows and number of columns from user and Display below pattern.

Input:  iRow=3   iCol=4
Output : 
         1 1 1 1
         2 2 2 2
	     3 3 3 3

*/     
		  
#include<stdio.h>

void Pattern(int iRow,int iCol)
{
	int i=0;
	int j=0;
    
    if(iRow<0)
	{
	   iRow=-iRow;
	}

	if(iCol<0)
	{
	   iCol=-iCol;
	}

	for(i=1;i<=iRow;i++)
	{  
       for(j=1;j<=iCol;j++)
	   {
		   printf("%d\t",i);
	   }
	   printf("\n");
    }
}

int main()
{
  int iValue1=0,iValue2=0;

  printf("Enter number of rows and columns");
  scanf("%d %d",&iValue1,&iValue2);

  Pattern(iValue1,iValue2);

  return 0;
}  	  

File: Java programs/Problems on Pattern Printing/Print square and number pattern/Pattern5.java
/* Write a java program which Accept the number of rows and number of columns from user and Display below patten.

Input:  iRow=3   iCol=4
Output : 1 1 1 1
         2 2 2 2
         3 3 3 3
         4 4 4 4       

*/
		  
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
	   int i=0;
	   int j=0;
	   for(i=1;i<=iRow;i++)
	   {  
          for(j=1;j<=iCol;j++)
	      {
		     System.out.print(i+"\t");
	      }
	     System.out.println();
       }   
	}   
}

class Lba375
{
    public static void main(String arg[])
    {
       Scanner sobj=new Scanner(System.in);

		int iValue1=0,iValue2=0;

        System.out.println("Enter number of Rows ");
        iValue1=sobj.nextInt();

		System.out.println("Enter number of Columns");
		iValue2=sobj.nextInt();

		Pattern pobj=new Pattern();
		
        pobj.Pattern(iValue1,iValue2);
	}	
}  	  
	  